### DISCUSSION TITLE
Crouton installation problem

### DISCUSSION THREAD

**Discussion by ghost** [dnschneid/crouton#4454] [Q&A]
Hello I have a problem installing crouton this message appears all the time and I can not fix it can you help me? 

:) 
![Screenshot 2021-04-11 00 09 30](https://user-images.githubusercontent.com/82291974/114285885-d9fe7c00-9a5a-11eb-866d-0ca5d074b3e6.png)

**Comment by DennisLfromGA** [comment]
This is also happening in xenial & bionic it appears -

- Help please. #4441
- crouton on arm, kernel version issue with libc updating in chroot · Issue #4429 · dnschneid/crouton
- sid installation issues on Chrome OS 90.0.4430.11 #4434

I believe it's outdated distros and there may not be anything that can be done. 🤔
